### ISSUE TITLE
Missing variants in local database?

### ISSUE THREAD

**Issue by 1ctw** [Clinical-Genomics/scout#1349]
I noticed a variant that is almost in all newly analyzed cases (7 hits in "All SNVs and INDELS"). It does however not appear in the local database. How often is the local database updated? It would maybe be good to do an update to get rid of those variants that are common now after Scout update.

https://scout.scilifelab.se/cust003/19199/6a961c3f0c5ade4f125c38e4241088e5

**Comment by dnil** [comment]
Ping @henrikstranneheim for a suggestion on what is practical! I would humbly advise to amass a few hundred cases for a freeze to make it count.

**Comment by henrikstranneheim** [comment]
This is related to the what we talked about in the Scout user meeting.
1. There were issues with duplications and cancer samples being present in the current rare disease loqusdb that Scout query.
2. A new one has been drafted, with a better more stringent implementation that removes the faults the old one had. However, that is empty.
3. Productions samples are now filling up the new better loqus implementation (current rate 1/h). The rate is set to not strain clinical-db more than necessary. You have already experienced the slowing down, when we reduced the allowed memory usage. However, this also means that the old loqusdb Scout is querying is not filling up any more.
4. Question is when to redirect Scout to the new loqusdb and when to make a first export. 

Suggestions on a suitable number of cases is most welcome.
